application/use_cases/stock_locations/delete_stock_location/idelete_stock_location_output_port.py

```diff
@@ -0,0 +1,20 @@
+from abc import ABC, abstractmethod
+
+from clapy import IOutputPort
+
+from domain.entities.base_entity import EntityID
+
+class IDeleteStockLocationOutputPort(IOutputPort, ABC):
+
+    @abstractmethod
+    async def stock_location_deleted_async(self, stock_location_id: EntityID) -> None: # Do i want to send out the ID?
```

Comment:
i have no idea what we want to come out of a delete yet, we may not need anything to come out